### ISSUE TITLE
Search not showing all results on the webclient (but Windows shows them)

### ISSUE THREAD

**Issue by BlepingMatrix** [airdcpp-web/airdcpp-webclient#390]
Okay I noticed something very odd. On the webclient when I search for a specific release (and only that release --the others work fine) I don't get any hits on the search. However, when done on Windows there is a result. I'm active mode on the webclient. The person testing on Windows is active mode on his end too. I don't understand why this is happening though. 

Note: I'm not connecting to the hub from Windows and Linux at the same time. I had a friend test it. He didn't get a hit over the webclient either. 

The user who has this release is marked "Away" MAYBE that's got something to do with it?

This is not the first time I've noticed it though. Its happened with two different users. I could be more specific in a direct message probably. Could y'all help?

Application version
AirDC++w 2.11.0 x86_64
Web UI version
2.11.0
Web UI build date
January 8, 2021 7:30 PM
API version
1
API feature level
7

**Comment by maksis** [comment]
You could launch the application with `--cdm-client` parameter. After performing the search, check the server console for `URES` messages to see whether results were received. 

If you can't see any messages, most likely those were not sent by the other user (this can be confirmed only by checking the outgoing client commands from his client) or the results got filtered by something else before they reached your client. In that case you could also try switching to passive mode to see if it makes any difference.

**Comment by BlepingMatrix** [comment]
Yeah I can't see any messages if I search that specific name, but messages show up for any other release. Thanks anyway I guess. (and I can't switch to passive mode or I'll get banned from the hub :D)

**Comment by maksis** [comment]
I'd really recommend checking the issue with the other guy if he's running AirDC++. In case he's using the Windows version, he can view the outgoing search results by opening the CDM debug window (`View` -> `CDM debug messages`) and ticking `Client commands (UDP)`. Note that window needs to be open before you perform the search.

I remember seeing some other users experiencing similar issues but no one has tried to investigate it further.

**Comment by BlepingMatrix** [comment]
I did yeah. If the other user isn't connectable this tends to happen but interestingly enough the result still shows up in the windows client but not the Linux one. (It's possible that the windows client previously downloaded and kept this person's filelist and was just searching that while you know how it is in the Linux ones. All partial temporary lists)

Sep 20, 2021, 21:34 by ***@***.***:
